hvplot/tests/testutil.py

```diff
@@ -7,6 +7,13 @@
 import panel as pn
 import pytest
 
+try:
+    import spatialpandas
+    import dask
+except ImportError:
+    spatialpandas = None
+    dask = None
```

Comment:
I think you might want to add spatialpandas/dask to the tests requirements.
https://github.com/holoviz/hvplot/blob/main/pyproject.toml#L73-L80

Reply:
Done